Able to set the Default Terminal to Powershell 7 in Windows Terminal

blusewill opened the pull request:
Using one button able to install Powershell 7 and set the default terminal into it.

https://github.com/ChrisTitusTech/winutil/assets/62820836/2995670a-c48c-4b51-a9f7-a23c1b9df170

Marterich reviewed (commented):
Also, if possible maybe revert winutil.ps1 and inputTweaks.xaml to the state before your compile.
Because those files are autogenerated anyways the only create a huge amount of bloat and changed lines in the PR

blusewill commented on config/tweaks.json:
```diff
@@ -2253,6 +2253,41 @@
+        Write-Host 'Powershell 7 is already installed.'
+      } else {
+        Write-Host 'Installing Powershell 7...'
+        winget install --id Microsoft.PowerShell --silent --accept-source-agreements --accept-package-agreements
```
> I think it would be better to utilize the function Install-WinUtilProgramWinget to install Microsoft.Powershell so we keep the different locations in the script where direct calls to winget are used minimal

![image](https://github.com/ChrisTitusTech/winutil/assets/62820836/ff78535d-d431-44f6-b03d-04c216b6ab06)

It's a me problem or I can't Invoke the Script into the File?

![image](https://github.com/ChrisTitusTech/winutil/assets/62820836/3329f4a5-58cc-4147-944e-87525d14a254)

blusewill commented:
> Also, if possible maybe revert winutil.ps1 and inputTweaks.xaml to the state before your compile.
> Because those files are autogenerated anyways the only create a huge amount of bloat and changed lines in the PR

I know. But I use Cli to commit all of the files
I know the file are autogenerated that's why I deleted in the Pull Request

blusewill commented:
Alright I made it using the Install-WinUtilProgramWinget But does anyone know how does this function work?

blusewill commented:
> I'll check the code later today. Will come back to you :)
> Edit: @blusewill I created a PR to your Repo with a few modifications that should work as expected 

Awesome will check that later today
